### ISSUE TITLE
Unhandled Runtime Error

### ISSUE THREAD

**Issue by EthanHook2** [prisma/prisma#20490]
### Bug description

```
Error: PrismaClient is unable to be run in the browser.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
```

Heres the contents of `lib/prisma.ts`
![code](https://github.com/prisma/prisma/assets/88622115/776b79bb-5e1c-4f95-957b-9a72f771c0b5)

### How to reproduce

I just refrehed my Next.js app after completing a feature and on every page of my web app im getting this error. the feature I was working on had nothing to do with prisma. 

### Prisma information

```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model competencys {
  id    Int     @id @default(autoincrement())
  name  String
  users users[] @relation("competencystousers")
}

model groups {
  id    Int     @id @default(autoincrement())
  name  String
  users users[] @relation("groupstousers")
}

model traininggroups {
  id    Int     @id @default(autoincrement())
  name  String
  users users[] @relation("traininggroupstousers")
}

model users {
  id                    Int              @id @default(autoincrement())
  email                 String           @unique(map: "Users_email_key")
  firstName             String
  lastName              String
  password              String
  dateOfBirth           DateTime         @map("date_of_birth")
  discordId             String?
  uses2fa               Boolean?
  gameLiscence          String?
  steamId               String?
  teamspeakId           String?
  accountCraeted        DateTime?
  lastLogin             DateTime?
  ipaddress             String?
  canapplyDate          DateTime?
  acceptanceDate        DateTime?
  loadendDate           DateTime?
  forceNumber           String?
  rank                  String?
  branch                String?
  status                String?          @default("new")
  authCode              String?
  twoFactorCode         String?
  acceptedPrivacyPolicy Boolean          @default(false)
  expirationTime        DateTime?
  competencys           competencys[]    @relation("competencystousers")
  groups                groups[]         @relation("groupstousers")
  traininggroups        traininggroups[] @relation("traininggroupstousers")
}
```

### Environment & setup

- OS: Windows
- Database: MySQL, 
- Next.jsv13 latest Canary
- Node.js version: v20.4.0

### Prisma Version

```
prisma                  : 5.1.0
@prisma/client     : 5.1.0

```

**Comment by janpio** [comment]
Thanks for the report @EthanHook2. Can you maybe help us with a more complete reproduction? What is the minimal setup we need to experience this problem?

**Comment by jharrell** [comment]
Additionally, could you try our latest dev branch to see if the problem is resolved? Not a long term solution, but will let us know if we’ve already resolved your issue in an upcoming patch. 

```

npm install -D prisma@5.2.0-integration-fix-enums-browser.1
npm install @prisma/client@5.2.0-integration-fix-enums-browser.1
npx prisma generate
```

**Comment by mellson** [comment]
@jharrell this version fixed the problem in our case

**Comment by EthanHook2** [comment]
> Additionally, could you try our latest dev branch to see if the problem is resolved? Not a long term solution, but will let us know if we’ve already resolved your issue in an upcoming patch.
> 
> ```
> 
> npm install -D prisma@5.2.0-integration-fix-enums-browser.1
> npm install @prisma/client@5.2.0-integration-fix-enums-browser.1
> npx prisma generate
> ```

I will go update my local branch to see if this fixes the issue, and get back to you.  Thank you

**Comment by EthanHook2** [comment]
From updating to that new dev version, its working a charm now my Next.jsv13 web app. Thank you

**Comment by AndreiOvercode** [comment]
> Additionally, could you try our latest dev branch to see if the problem is resolved? Not a long term solution, but will let us know if we’ve already resolved your issue in an upcoming patch.
> 
> ```
> 
> npm install -D prisma@5.2.0-integration-fix-enums-browser.1
> npm install @prisma/client@5.2.0-integration-fix-enums-browser.1
> npx prisma generate
> ```

I had the same problem and this solved it

**Comment by janpio** [comment]
Thanks for your feedback, we will release this fix as 5.1.1 tomorrow. Stay tuned.

**Comment by Jolg42** [comment]
I think we can close this as a duplicate of https://github.com/prisma/prisma/issues/20480
And we'll let you know tomorrow when the patch is out 🙌🏼
